### DISCUSSION TITLE
where store the sevice name and instance name list?

### DISCUSSION THREAD

**Discussion by taoyang987** [apache/skywalking#11866] [Q&A]
i explore the log data with kibana from sw_log index.
in the index ,there is only sevice id and instance
where store the sevice name and instance name list?
if the name filed also in the index ,it will be very readable from kinbana
![WX20240203-151214@2x](https://github.com/apache/skywalking/assets/872386/87db9425-8db5-4fbf-a365-1f55f4e2edda)

**Comment by wu-sheng** [comment]
We don't build data for the kibana.
We also suggest you to use official query APIs.

**Reply by taoyang987** [reply]
my boss told me to use sw for parse and analyze log instand of elk.that's the target .we promise our customer using sw to get apm and log alert.

**Reply by wu-sheng** [reply]
That is your supervisor, not ours.
Don't put the things in a wrong way.
